physionet-django/physionet/views.py

```diff
@@ -1,20 +1,26 @@
+from django.contrib.auth.decorators import permission_required, login_required
```

Comment:
Imports are ordered into groups (1. python standard library, 2. external packages, 3. local imports) and broadly alphabetical. The django import should appear below at ~line 10.

Reply:
thanks @tompollard, the order of imports has been fixed.